### PR TITLE
Add video transcription to transition landing page

### PR DESCRIPTION
**Pull request by hannako**
## What
Add transcription of the youtube video to the transition landing page. 

## Review app
https://govuk-collec-video-tran-ceuh41.herokuapp.com/transition

Screenshot 2020-07-21 at 14 23 27
Screenshot 2020-07-21 at 14 23 37

Trello: https://trello.com/c/qDJkX4wa/325-add-video-transcript-to-landing-page-english

### CODE REVIEW

**Comment by hannako** [comment]
@DilwoarH could i get your help on making the arrow white!

**Comment by DilwoarH** [comment]
Hey @hannako 
```css

.govuk-details__summary {
  color: govuk-colour('white');
}

.govuk-details__text {
  color: govuk-colour('white');
}

```

**Comment by DilwoarH** [comment]
> Hey @hannako
> 
> ```css
> .govuk-details__summary {
>   color: govuk-colour('white');
> }
> 
> .govuk-details__text {
>   color: govuk-colour('white');
> }
> ```

You might want to add modifier classes. If you have issues - I can pull down the branch and do it

File: app/views/transition_landing_page/components/_youtube_player.html.erb [hunk 1]
```diff
@@ -33,4 +33,14 @@
       </a>
     </div>
   <% end %>
+  <details class="govuk-details" data-module="govuk-details">
```
Comment (andysellick) [inline app/views/transition_landing_page/components/_youtube_player.html.erb:36]: Could you use the [details component](https://components.publishing.service.gov.uk/component-guide/details) for this?

**Comment by hannako** [comment]
@andysellick @DilwoarH What do you think the hover state should be https://govuk-collec-video-tran-ceuh41.herokuapp.com/transition ?

**Comment by andysellick** [comment]
> @andysellick @DilwoarH What do you think the hover state should be https://govuk-collec-video-tran-ceuh41.herokuapp.com/transition ?

The only other link on that blue background is the language switcher at the top of the page, which has no hover state. Suggest going with that - the blue doesn't lend itself well to any other colour I can think of for the hover state.

@DilwoarH I'd suggest doing an inverse option for the details component - overriding the styles like this breaks component isolation.

**Comment by hannako** [comment]
Thanks @andysellick have updated the hover state as you suggest 👍 

Could we get this change live, and then update the component afterwards?

File: app/assets/stylesheets/components/_youtube-player.scss [hunk 2]
```diff
@@ -39,3 +39,20 @@
...
 }
+
+.govuk-details__summary {
+  color: govuk-colour('white');
+  &:hover {
+    color: govuk-colour('white');
+  }
+}
+
+.govuk-details__text {
```
Comment (danacotoran) [inline app/assets/stylesheets/components/_youtube-player.scss:50]: Nickpick could we please also add `border-left-color: govuk-colour('white');`  for `.govuk-details__text` –  looks like the border on the left is white in the mockup, and it looks slightly off as a grey colour against the blue background

Comment (hannako) [reply app/assets/stylesheets/components/_youtube-player.scss:50]: Thanks Dana! Will do.

File: app/assets/stylesheets/components/_youtube-player.scss [hunk 3]
```diff
@@ -39,3 +39,20 @@
     border-radius: 100%;
   }
 }
+
+.govuk-details__summary {
```
Comment (danacotoran) [inline app/assets/stylesheets/components/_youtube-player.scss:43]: Nitpick could we please add
```
.govuk-details__summary {
  color: govuk-colour('white');
  &:hover {
    color: govuk-colour('white');
  }
  &:hover:focus,
  &:hover:focus .govuk-details__summary-text {
    color: govuk-colour('black');
  }
}
```

This should ensure that the text and arrow are legible when in focus. With the current styling, if the element is in focus the text turns white on a yellow background. This is a WCAG colour contrast fail.

The solution above is not 100% ideal but it should do in order to make it look decent for now 😕

Comment (hannako) [reply app/assets/stylesheets/components/_youtube-player.scss:43]: Thanks Dana

File: app/assets/stylesheets/components/_youtube-player.scss [hunk 4]
```diff
@@ -45,10 +45,15 @@
...
   }
+  &:hover:focus,
+  &:hover:focus .govuk-details__summary-text {
+    color: govuk-colour('black');
+  }
 }
 
 .govuk-details__text {
   color: govuk-colour('white');
+  border-left-color: govuk-colour('white')
```
Comment (danacotoran) [inline app/assets/stylesheets/components/_youtube-player.scss:56]: Could we add a semicolon at the end of this line please (sorry!)

Comment (hannako) [reply app/assets/stylesheets/components/_youtube-player.scss:56]: Sorry! It's been a while since I made any changes to scss files :-)

**Pull request merged**